data/mods/Aftershock/maps/furniture_and_terrain/furniture_habitat.json

```
@@ -1,4 +1,50 @@
 [
+  {
+    "type": "furniture",
+    "id": "f_gene_editor",
+    "name": "Mercurial Resequencer",
+    "description": "A large bio-sarcophagus and accompanying control machinery.  Used in gene editing and for general medical purposes, can impart or remove traits using a variety og gene templates.",
```

```suggestion
    "description": "A large bio-sarcophagus and accompanying control machinery.  Used in gene editing and for general medical purposes, can impart or remove traits using a variety of gene templates.",
```
I noticed the spell check was complaining about this one.